src/agor/tools/memory_manager.py

```
@@ -85,11 +85,11 @@ def commit_to_memory_branch(
     print("🛡️  Safe memory commit: staying on current branch")
 
     # Get current branch to stay on it
-    success, current_branch = run_git_command(["branch", "--show-current"])
+    success, current_branch_output = run_git_command(["branch", "--show-current"])
     if not success:
-        print("❌ Cannot determine current branch")
+        print(f"❌ Cannot determine current branch. Error: {current_branch_output}")
         return False
-    current_branch = current_branch.strip()
+    current_branch = current_branch_output.strip()
```

Ruff (0.11.9)

Local variable `current_branch` is assigned to but never used

Remove assignment to unused variable `current_branch`

(F841)